network/f5/bigip_irule_bigsuds.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
```

Comment:
Copyright & license notice missing. You can copy this from one of the existing modules